code/figure5.jl

```diff
@@ -0,0 +1,68 @@
+include("seed.jl")
+
+# Simulation with specialist natural enemies with stochasticity on D
+sim1, params1 = simulation(50.0, 25.0, m=0.2, F=4.0, D=0.5, c=1.0, a=0.5, th= 0.0, m=0.5, f=specialist_dyn, D_std=0.5)
```

Comment:
Some of the function calls are still passing in every parameter, even if it's equal to the default value. This makes it very difficult to see what's actually changed from the baseline value in this line. I am looking for if two parameters were accidentally changed which caused the discrepancy in the figure, but it's hard to tell from this version of the code.